projects/packages/status/src/class-status.php

```diff
@@ -171,11 +171,12 @@ public function is_local_site() {
 	/**
 	 * If is a staging site.
 	 *
-	 * @todo Add IDC detection to a package.
+	 * @deprecated since $$next-version$$
 	 *
 	 * @return bool
 	 */
 	public function is_staging_site() {
+		_deprecated_function( __FUNCTION__, '$$next-version$$', 'in_safe_mode' );
 		$cached = Cache::get( 'is_staging_site' );
```

Comment:
Could you remove the contents of the function here (apart from the `_deprecated_function` line) and just call `in_safe_mode` from it instead, perhaps?

Reply:
I think I could remove all of it, but then I would need to return `in_safe_mode()`. Since those two methods are not quite the same, I went with this option, which would recommend using `in_safe_mode()` but not forcing it. Does that make sense?

Reply:
Sure, makes sense if they're not quite the same.